pkgs/development/python-modules/grep-ast/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, python3
```

Comment:
Just to make sure I understand, you're suggesting I move all packages imports to the top? Should I do the same for `.../aider/package.nix`?

Reply:
No it was just for this package, because it'd get called with `python3Packages.callPackage` not just `callPackage` - but I'll mark this resolved if we'se going with the other PR for grep-ast.